INSTALL

```diff
@@ -1,5 +1,31 @@
-Installing Iris currently requires you to build it from source. The 
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
```

Comment:
The reason for the markup is that this file is explicitly included in 
https://raw.githubusercontent.com/SciTools/iris/master/docs/iris/src/installing.rst
and built into the docs

I think it should stay to maintian this docs page